.github/ISSUE_TEMPLATE/support.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
It has been suggested by @e0ne that this may not be required as a bug template may be used instead.
Upside of having this is that this template will force the issuer to read the comment below before opening.

Im fine either way (remove/keep).

Reply:
I don't get this comment. is it on the "Support Request" template?

Reply:
yes, the comment is whether or not we need this template.

Reply:
I will remove it.

Reply:
This issue will make the process more complicated. IMO, feature requests and bug reports should be enough.

Reply:
right, we agree to remove it